metricbeat/module/ceph/health/data.go

```diff
@@ -0,0 +1,119 @@
+package health
+
+import (
+	"encoding/json"
+	"github.com/elastic/beats/libbeat/common"
+	"github.com/elastic/beats/libbeat/logp"
+	"io"
```

Comment:
we normally but the external imports on the two following and empty line before the internal once